src/workerd/api/streams/readable.h

```diff
@@ -142,6 +146,11 @@ class ReadableStreamBYOBReader: public jsg::Object,
 
     JSG_METHOD(readAtLeast);
     // Non-standard extension that should only apply to BYOB byte streams.
+
+    JSG_TS_OVERRIDE(ReadableStreamBYOBReader {
+      read<T extends ArrayBufferView>(view: T): Promise<ReadableStreamReadResult<T>>;
+      readAtLeast(minBytes: number, view: Uint8Array): Promise<ReadableStreamReadResult<Uint8Array>>;
```

Comment:
For read at least, let's make the first argument `atLeast` or just `min`. With a recent change (landed last week), that first argument is the number of distinct elements in the `view`, which can now be any `TypedArray`. For instance `readAtLeast(2, new Uint32Array(2))` means that the minimum number of bytes to be read is 8.